src/commands/OSRS_Fun/dryCalc.ts

```diff
@@ -0,0 +1,45 @@
+import { Command, CommandStore, KlasaMessage } from 'klasa';
+
+/* 
+  Witty Switch Comments Pulled From:
+  https://oldschool.runescape.wiki/w/Calculator:Dry_calc
+*/
+export default class extends Command {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 1,
+			description:
+				'Calculates the drop chance of getting an item within a set amount of rolls.',
+			usage: '<dropRate:int> <rolls:int>',
+			usageDelim: ' '
+		});
+	}
+
+	async run(msg: KlasaMessage, [dropRate = 1, rolls = 1]: [number, number]) {
+		if (dropRate > 100000000 || rolls > 100000000) {
+			throw "I can't process a number higher than 100 million! enneUni has already claimed the only uncut onyx!";
+		}
+		if (dropRate < 1 || rolls < 1) {
+			throw "I can't calculate with zeros nor negative numbers. Do more rolls or update the drop values.";
+		}
+		const noDropChance = Math.pow(1 - 1 / dropRate, rolls);
+		const dropChance = 100 * (1 - noDropChance);
```

Comment:
i dont see the reason to include this, its just the inverse of how dry you are, and this command is called `drystreak`

Reply:
it was just the way the oldschool bot drycalc was showing so I figured that's what most ppl would prefer. For some the inverse is obvious but not for everyone so wanted to make sure that it includes a greater auidience.